packages/nextjs/src/config/webpack.ts

```diff
@@ -195,37 +192,34 @@ function addFileToExistingEntryPoint(
   filepath: string,
 ): void {
   // can be a string, array of strings, or object whose `import` property is one of those two
-  let injectedInto = entryProperty[entryPointName];
-
-  // Sometimes especially for older next.js versions it happens we don't have an entry point
-  if (!injectedInto) {
-    // eslint-disable-next-line no-console
-    console.error(`[Sentry] Can't inject ${filepath}, no entrypoint is defined.`);
-    return;
-  }
+  const currentEntryPoint = entryProperty[entryPointName];
+  let newEntryPoint: EntryPointValue;
 
   // We inject the user's client config file after the existing code so that the config file has access to
   // `publicRuntimeConfig`. See https://github.com/getsentry/sentry-javascript/issues/3485
-  if (typeof injectedInto === 'string') {
-    injectedInto = [injectedInto, filepath];
-  } else if (Array.isArray(injectedInto)) {
-    injectedInto = [...injectedInto, filepath];
-  } else {
-    let importVal: string | string[];
-
-    if (typeof injectedInto.import === 'string') {
-      importVal = [injectedInto.import, filepath];
+  if (typeof currentEntryPoint === 'string') {
+    newEntryPoint = [currentEntryPoint, filepath];
+  } else if (Array.isArray(currentEntryPoint)) {
+    newEntryPoint = [...currentEntryPoint, filepath];
+  }
+  // descriptor object (webpack 5+)
+  else {
```

Comment:
Can we explicitly check for an object and just throw an error if we get something else? Will help debugging in the future in the case they change this.

Reply:
Done.